faker/providers/address/en_US/__init__.py

```diff
@@ -329,8 +333,17 @@ def secondary_address(self):
     def state(self):
         return self.random_element(self.states)
 
-    def state_abbr(self):
-        return self.random_element(self.states_abbr)
+    def state_abbr(self, include_territories=True):
+        """
+        :returns: A random state or territory abbreviation.
+
+        :param include_territories: If True, territories will be included.
+            If False, only states will be returned.
+        """
+        abbr_options = self.states_abbr
+        if include_territories:
+            abbr_options += self.territories_abbr
```

Comment:
This method could potentially be called thousands of times, and I'm worried about the performance of creating the list at every call. I would rather define another class constant like `states_with_territories_abbr = states_abbr + territories_abbr` and have
```python
if include_territories:
    return self.random_element(states_with_territories_abbr)
```

Reply:
That makes sense. Thanks for reviewing my pull request! :) I will get a new commit in shortly to address that.